src/Tabs/Tab.js

```diff
@@ -47,22 +48,31 @@ export const styleSheet = createStyleSheet('MuiTab', (theme) => {
     fullWidth: {
       flex: '1 0 0',
     },
+    labelContainer: {
+      paddingTop: 6,
+      paddingBottom: 6,
+      paddingLeft: 12,
+      paddingRight: 12,
+      [theme.breakpoints.up('md')]: {
+        paddingLeft: 24,
```

Comment:
Thanks for the detail though. Yeah, I think that consistency and clean abstraction is a good one to follow that direction, but as you said, I'm wondering if that has any use cases for users.